src/utils/Hook/stories.tsx

```diff
@@ -17,15 +17,14 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+import { action } from '@storybook/addon-actions';
 import { storiesOf } from '@storybook/react';
 import React from 'react';
-import { action } from '@storybook/addon-actions';
-
-import Button from '../../Button';
 import Hook from '.';
+import { Button } from '../../Button';
 import readme from './readme.md';
 
-const HookStories = storiesOf(`${__dirname}`, module).add(
+storiesOf(`Hook`, module).add(
   'Basic',
   () => (
     <Hook
```

Comment:
Failing w:
`Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

Reply:
import './' vs import './index' fixed